src/query/server/query.go

```diff
@@ -640,45 +640,40 @@ func newM3DBStorage(
 		namespaces  = clusters.ClusterNamespaces()
 		downsampler downsample.Downsampler
 	)
-	if n := namespaces.NumAggregatedClusterNamespaces(); n > 0 {
```

Comment:
Diff is weird here, but this change is just me removing the conditional on this line.